src/classes/ALLO_Allocations_TEST.cls

```diff
@@ -37,6 +37,53 @@
 @isTest
 private with sharing class ALLO_Allocations_TEST {
 
+    // These variables are used in a "Setup Results" object map to find different
+    // test data that was created
+
+    /** @description Key to find GAUs with */
+    private static final String GAUS_KEY {
```

Comment:
In a unit test context, is this any different than the following?
`private static final String GAUS_KEY = 'GAUS'`?

Reply:
No, not really.  I was being a little over obsessive about lazy initialization for class variables to keep heap allocation low...but it really isn't necessary for these, so I will change them.